ddtrace/internal/runtime/runtime_metrics.py

```diff
@@ -78,7 +79,7 @@ def flush(self):
         for key, value in self._runtime_metrics:
             self._write_metric(key, value)
 
-    on_periodic = flush
+    run_periodic = flush
```

Comment:
🤦‍♂ 

I guess the tests passed because we call `flush` manually?

Reply:
yeah, we call `flush` on `stop()`

Reply:
or `join` or w/e